great_expectations/core/util.py

```diff
@@ -425,10 +412,10 @@ def convert_to_json_serializable(  # noqa: C901 - complexity 32
         return data.to_json_dict()
 
     # PySpark schema serialization
-    if StructType is not None and isinstance(data, StructType):
+    if isinstance(data, sparktypes.StructType):
```

Comment:
I think this needs to make sure sparktypes is imported
```suggestion
    if sparktypes and isinstance(data, sparktypes.StructType):
```
or better do the same check with pyspark_StructType from optional imports

Reply:
@anthonyburdi I ned to fix this one -- good catch.

Reply:
@anthonyburdi Fixed (committed).  Thanks!